Add blog post about May 26th plugin update

azigler opened the pull request:
This PR adds a blog post about this week's `mattermost-plugin-ai` update summarized here:  https://community.mattermost.com/core/pl/ih9qbrx6ipfj9r6pcgbmdmna4w

azigler committed:
Add blog post about May 26th plugin update

jasonblais commented on content/posts/may-26-plugin-update.md:
```diff
@@ -0,0 +1,19 @@
+
+🔠 **Font Reverted to Regular**: We have reverted from using a monospaced font to a regular font. However, don't worry! We are working on introducing additional controls to differentiate the AI Assistant from regular users so there is no confusion. 
+
+AI open source technologies continue to evolve, empowering us to create intelligent and collaborative tools that augment human capabilities. Let's embrace the potential of AI and open source to shape a future where intelligent assistants empower us to achieve greater productivity and success.
```
@azigler what are your thoughts on adding a CTA, e.g. to the GitHub repo or the community channel?

azigler replied:
@jasonblais Great idea! I updated the blog post to use @asaadmahmood's new metadata format, added CTAs for the channel and forums, and changed the repo name to be a link to GitHub. I updated the date to today and attributed it to @crspeller.

azigler committed:
✨ feat(posts): add new post about May 26th update to `mattermost-plugin-ai`
This commit adds a new blog post about the May 26th update to the `mattermost-plugin-ai`. The update includes a new interface for the AI Assistant, which has been renamed from LLM Bot to AI Assistant for better clarity. The AI Assistant now has a new profile icon and enhanced feedback reactions. The font has been reverted to regular, but additional controls will be introduced to differentiate the AI Assistant from regular users. The post encourages the community to embrace the potential of AI and open source to shape a future where intelligent assistants empower us to achieve greater productivity and success.

azigler commented:
@asaadmahmood Can you merge when you have a chance?

asaadmahmood commented:
@jasonblais @azigler We need to run `hugo -d docs` before pushing any changes.

azigler committed:
Build docs

azigler commented:
> @jasonblais @azigler We need to run `hugo -d docs` before pushing any changes.

Gotcha. I went ahead and did that now. @jasonblais should we consider setting up some automations for this process?

asaadmahmood commented:
I have pushed this separately to avoid the conflicts.

azigler commented:
@asaadmahmood Looks good but seems like the Markdown for `inline code` is not being rendered in the title:

https://ai.mattermost.com/posts/2023-05-30-plugin-update/

The title here (and on the homepage) is "May 26th update to \`mattermost-plugin-ai\`"

asaadmahmood commented:
@azigler I'm not sure if the title allows the markdown to render.

azigler commented:
@asaadmahmood Gotcha. If so, can you remove them from that title when you get a chance?

asaadmahmood commented:
@azigler Done.
